SPECS/mariner-rpm-macros/gen-ld-script.sh

```diff
@@ -9,8 +9,8 @@
 # /usr/lib/rpm/mariner/gen-ld-script.sh %{name} %{version}
 echo "gen-ld-script.sh name($1) version($2)"
 
-OS_ID=$(grep -oP '(?<=^ID=).+' /etc/os-release | tr -d '"')
-OS_VERSION=$(grep -oP '(?<=^VERSION_ID=).+' /etc/os-release | tr -d '"')
+OS_ID=$(sed -En '/^ID=/ s/.+="?([^"]+)"?/\1/ p' /etc/os-release)
+OS_VERSION=$(sed -En '/^VERSION_ID=/ s/.+="?([^"]+)"?/\1/ p' /etc/os-release)
```

Comment:
What are these changes for?

Reply:
(The functionality stays the same)

Reply:
Here was the error:
grep: support for the -P option is not compiled into this --disable-perl-regexp binary